app/src/main/java/com/dinodevs/pacecalendarwidget/APtranslations.java

```diff
@@ -42,7 +42,7 @@
                 {"воскресенье", "понедельник", "вторник", "среда", "четверг", "пятница", "суббота"},
                 {"Nedeľa", "Pondelok", "Utorok", "Streda", "Štvrtok", "Piatok", "Sobota"},
                 {"Domingo", "Lunes", "Martes", "Miércoles", "Jueves", "Viernes", "Sábado"},
-                {"อาทิตย์", "จันทร์", "อังคาร", "พุธ", "พฤหัสบดี", "ุกร์", "สาร์"},
+                {"อา", "จ", "อ", "พ", "พฤ", "ศ", "ส"},
```

Comment:
This seems to be only the short version of the days (something like the english M T W T F S S).

Reply:
The full name of friday: "ุกร์", saturday: "สาร์" are wrong, The correct one are "ศุกร์" and "เสาร์".

It's too wide in the narrow collumn.
The first letter in small space calendar is pretty standard here in Thailand (:

Reply:
Yes, day first letter in calendars is standard here in Greece too.
My question was, if we can correct the translation to
`{"อาทิตย์", "จันทร์", "อังคาร", "พุธ", "พฤหัสบดี", "ศุกร์", "เสาร์"}`
and then the java code can extract the front letters needed.

For example, do these make sense? Or are they wrong?:

| Thai | English |
| ------------- | ------------- |
| อ, จั, อั, พุ, พ, ศุ, เ | S, M, T, W, T, F, S |
| อา, จัน, อัง, พุธ, พฤ, ศุก, เส | Su, Mo, Tu, We, Th, Fr, Sa |

Reply:
I got it...
No it not make sense, the meaning will be difference 
Ex. 
เส = pronounce as 'Se' maening slip
เสา pronounce as 'Soa' but this word mean a pole.
เสาร์ pronounce as 'Soa' is Saturday

the simply substring method is not sufficient.
you need through and discard these เ, า, -ั, -ุ, -ิ they are not consonant.
so you will get อ, จ, อ, พ, พ, ศ and ส
sound like a lots of work haha.

sorry for my English.

Reply:
Ok. From my point of view there are 2 solutions, the first solution is to leave only the short 1 letter version (your implementation), the other solution is to implement an exception system for such cases as this problem probably exists on other languages too.

I think for now, @GreatApo should pull your commit.